torch/_refs/nn/functional/__init__.py

```diff
@@ -233,6 +236,36 @@ def selu(a: TensorLikeType, inplace: bool = False) -> TensorLikeType:
     return scale * torch.where(a > 0, a, rhs)
 
 
+# CompositeImplicitAutograd - don't register decomp
+def softmax(
+    a: TensorLikeType,
+    dim: Optional[int] = None,
+    _stacklevel: int = 3,  # for compat when using TorchRefsMode(strict=True)
+    dtype: Optional[torch.dtype] = None,
+) -> TensorLikeType:
+    # The error is for compat with regular PyTorch, which has this behavior
+    # deprecated.  For PrimTorch, it's fine to drop support for deprecated
+    # behavior because it requires explicit opt in.  This error is to inform
+    # users how to update their calls.
+    check(dim is not None, lambda: "implicit dim not supported, use dim=X")
+    return torch.softmax(a=a, dim=dim, dtype=dtype)  # type: ignore[call-overload]
```

Comment:
This reference basically assumes a softmax prim -- but should softmax actually be a prim? 

If we want to do this to unblock something we should file an issue and comment here that this isn't actually going to a prim, and the hard work of implementing softmax remains.

Reply:
This reference doesn't assume softmax prim, it is just an alias.

Reply:
Aha! Thank you for the clarification